.travis.yml

```diff
@@ -1,8 +1,22 @@
 language: python
-python:
- - "2.7"
- - "3.6"
-install: pip install -r requirements.txt -r test-requirements.txt
-script:
-    - flake8 --filename="*.py"
-    - nosetests
+before_install:
+  - pip install wheel
```

Comment:
Why are we installing wheel specially?

Reply:
Ummm, I thougth it was necesary for wheels to be installed but Im not sure about that, I dont think so, its only for *creating* them

Let me remove that!